app/controllers/api/customers_controller.rb

```diff
@@ -33,5 +33,9 @@ def add_recurring_payment_info(customer, client_secret)
       customer.gateway_recurring_payment_client_secret = client_secret
       customer.gateway_shop_id = customer.enterprise.stripe_account&.stripe_user_id
     end
+
+    def customer_params
```

Comment:
The commit message says "remove whitespace" but you are adding a method you forgot in a previous commit, right?

Reply:
Yeah that was part of a messy merge conflict with the Rails 4.2 strong params stuff. I was adding it back after resolving the conflict... not sure why it has that message attached. I'll try cleaning it up as part of Operation Squash.

Reply:
I've updated the commit message 👍